android/app/src/main/res/menu/first_menu.xml

```diff
@@ -2,22 +2,6 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this file needed?

Reply:
Yes, it's the icons that appear in the action bar.